### ISSUE TITLE
[JENKINS-62437] Change Jenkins configuration UI from tables to divs

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/jenkins#11553]
<p>We would like to make Jenkins usable by as many people as possible. It includes multiple groups of users: people with disabilities, ones using mobile devices, or those with slow network connections. In general, all Jenkins users would benefit from better navigation and layouts. Currently the Jenkins's UI includes a lot of tables on configuration pages. These tables are very mobile hostile. The UI is also not in line with modern UIs.</p>
<h2><a name="Testing"></a>Testing</h2>
<ul>
	<li>Quick-start: <a href="https://github.com/oleg-nenashev/jenkins-tabs-to-divs-config-migration-testenv" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/oleg-nenashev/jenkins-tabs-to-divs-config-migration-testenv</a></li>
	<li>Manual setup: <tt>docker run --rm -ti -p 8080:8080 -e ID=3895 jenkins/core-pr-tester</tt></li>
</ul>

<h2><a name="Examples"></a>Examples</h2>

<p>Here's one example from Windows 10's settings:</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51287/51287_image-2019-02-12-15-44-44-739.png" height="453" width="372" style="border: 0px solid black" /></span></p>

<p>Note that the label for a field is above the field – not to the left of the field.</p>

<p>Here's a much older UI from Windows:</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51288/51288_image-2019-02-12-15-45-51-351.png" height="375" width="369" style="border: 0px solid black" /></span></p>

<p>Note that the labels for checkboxes/radios are to the right of the buttons, not to their left (* with RTL layouts, this changes, but that's due to mirroring and applies to almost everything). Even so, labels for other things are above, not to their left (see this example for invitation duration – which is also pre Windows 10):</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51289/51289_image-2019-02-12-15-46-51-006.png" height="243" width="349" style="border: 0px solid black" /></span></p>

<p> </p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/oleg_nenashev.png" title="oleg_nenashev's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=oleg_nenashev">oleg_nenashev</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-62437" target="_blank">Change Jenkins configuration UI from tables to divs</a></i></summary>
<i><ul>
<li><b>status</b>: In Progress
<li><b>priority</b>: Major
<li><b>component(s)</b>: core
<li><b>label(s)</b>: accessibility, roadmap, ux, ux-sig
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 3
<li><b>watchers</b>: 5
<li><b>imported</b>: 2025-11-24
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>We would like to make Jenkins usable by as many people as possible. It includes multiple groups of users: people with disabilities, ones using mobile devices, or those with slow network connections. In general, all Jenkins users would benefit from better navigation and layouts. Currently the Jenkins's UI includes a lot of tables on configuration pages. These tables are very mobile hostile. The UI is also not in line with modern UIs.</p>
<h2><a name="Testing"></a>Testing</h2>
<ul>
	<li>Quick-start: <a href="https://github.com/oleg-nenashev/jenkins-tabs-to-divs-config-migration-testenv" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/oleg-nenashev/jenkins-tabs-to-divs-config-migration-testenv</a></li>
	<li>Manual setup: <tt>docker run --rm -ti -p 8080:8080 -e ID=3895 jenkins/core-pr-tester</tt></li>
</ul>

<h2><a name="Examples"></a>Examples</h2>

<p>Here's one example from Windows 10's settings:</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51287/51287_image-2019-02-12-15-44-44-739.png" height="453" width="372" style="border: 0px solid black" /></span></p>

<p>Note that the label for a field is above the field – not to the left of the field.</p>

<p>Here's a much older UI from Windows:</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51288/51288_image-2019-02-12-15-45-51-351.png" height="375" width="369" style="border: 0px solid black" /></span></p>

<p>Note that the labels for checkboxes/radios are to the right of the buttons, not to their left (* with RTL layouts, this changes, but that's due to mirroring and applies to almost everything). Even so, labels for other things are above, not to their left (see this example for invitation duration – which is also pre Windows 10):</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51289/51289_image-2019-02-12-15-46-51-006.png" height="243" width="349" style="border: 0px solid black" /></span></p>

<p> </p></pre>
</details>
</details>
<details><summary><i>3 attachments</i></summary>

- [image-2019-02-12-15-44-44-739.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51287/image-2019-02-12-15-44-44-739.png)
  > ![image-2019-02-12-15-44-44-739.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51287/image-2019-02-12-15-44-44-739.png)
- [image-2019-02-12-15-45-51-351.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51288/image-2019-02-12-15-45-51-351.png)
  > ![image-2019-02-12-15-45-51-351.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51288/image-2019-02-12-15-45-51-351.png)
- [image-2019-02-12-15-46-51-006.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51289/image-2019-02-12-15-46-51-006.png)
  > ![image-2019-02-12-15-46-51-006.png](https://raw.githubusercontent.com/jenkinsci/attachments-from-jira-issues-core-cli/refs/heads/main/attachments/51289/image-2019-02-12-15-46-51-006.png)
</details>
